content/hi/docs/concepts/overview/working-with-objects/owners-dependents.md

```diff
@@ -0,0 +1,22 @@
+---
+title: ओनर और डिपेंडेंट 
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+कुबेरनेट्स में, कुछ {{< glossary_tooltip text="objects" term_id="object" >}} अन्य वस्तुओं के *ओनर* हैं। 
+उदाहरण के लिए, यह {{<glossary_tooltip text="ReplicaSet" term_id="replica-set">}} पॉड्स के एक सेट का ओनर है। ये ओन्ड वाली वस्तुएँ अपने ओनर पे *डिपेंडेंट* हैं।
```

Comment:
These localized lines have a slightly uneven flow, possibly attributed to machine translation. Consider refining the flow and substituting 'ऑब्जेक्ट' for 'वस्तु' for better coherence.

Reply:
added